public/themes/g.css

```diff
@@ -163,7 +163,7 @@ tr.gtr1 {
     height: 21px;
     margin: 4px 1px 0;
     padding: 0 4px 1px;
-    min-width: 150px;
+    min-width: 100px;
```

Comment:
A lot of the UI in the app uses .stdbtn and therefore changing the width across all of it to 100px might break a bunch of layouts.  
I'd prefer a dedicated .searchbtn complement class that overrides the min-width here.